box/session_test.go

```diff
@@ -0,0 +1,28 @@
+package box
```

Comment:
Please, test only the public API.
```suggestion
package box_test
```